mohawk/util.py

```diff
@@ -268,3 +267,37 @@ def normalize_header_attr(val):
     # Normalize like the hawk reference code.
     val = escape_header_attr(val)
     return val
+
+
+def get_bewit(resource):
```

Comment:
the semantics of `resource.timestamp` currently is more like "the time of the request (or response)"